test/e2e/test_log_types.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+
+import layer
+from layer.clients.layer import LayerClient
+from layer.contracts.logged_data import LoggedDataType
+from layer.contracts.projects import Project
+from layer.decorators import dataset
+from test.e2e.assertion_utils import E2ETestAsserter
+
+
+def test_scalar_values_logged(
+    initialized_project: Project, asserter: E2ETestAsserter, client: LayerClient
+):
+    # when
+    layer.init(initialized_project.name)
+
+    dataset_name = "scalar_ds"
+
+    # given
+    @dataset(dataset_name)
+    def scalar():
+        data = [[1, "product1", 15], [2, "product2", 20], [3, "product3", 10]]
+        dataframe = pd.DataFrame(data, columns=["Id", "Product", "Price"])
+        layer.log(
+            {
+                "str-log": "bar",
+                "int-log": 123,
```

Comment:
add bool and float too

Reply:
I actually had them, and even scientific notation. Just forgot to Put them back after merging with the existing test that I initially didn't notice and that was written by someone else. Will do!

Reply:
Done.